roles/cdi/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 cdi_namespace: "{{ cdi_image_namespace | default('golden-images') }}"
 cdi_storageclass: "{{ cdi_kubevirt_storageclass | default('kubevirt') }}"
 apb_action: "provision"
-repo_tag: "{{ cdi_repo_tag | default('jcoperh') }}"
+repo_tag: "{{ cdi_repo_tag | default('kubevirt') }}"
 release_tag: "{{ cdi_release_tag | default('latest') }}"
```

Comment:
release tags should reflect the most recent kubevirt version.  The `:latest` tag is deliberately not used.  Instead, CDI tracks kubevirt versions so as to ensure new kubevirt versions always use the most recent compatible version of CDI.